wifiphisher/pywifiphisher.py

```diff
@@ -77,6 +77,11 @@ def parse_args():
         "--no-mac-randomization",
         help=("Do not change any MAC address"),
         action='store_true')
+    parser.add_argument(
```

Comment:
All these arguments are great but a user won't find them unless you document them in README.md. Please add them there too!

Reply:
Done!